frontend/src/components/marketing/marketing.css

```
@@ -449,51 +651,368 @@
+  gap: var(--spacing-5);
+}
+
+.ppm-page .ppm-fitchef-reveal-card.ppm-fitchef-reveal-card--empty {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  min-height: 0;
+  padding: 0;
+  overflow: hidden;
+  border: 0;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)